ReactWindows/ReactNative/Modules/Vibration/VibrationModule.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using ReactNative.Bridge;
+using Windows.Phone.Devices.Notification;
+
+namespace ReactNative.Modules.Vibration
+{
+    /// <summary>
+    /// Represents the module for vibration.  Note this is only available on Windows Mobile.
+    /// </summary>
+    public class VibrationModule : NativeModuleBase
+    {
+        private readonly bool _isMobile;
+
+        internal VibrationModule() : this(IsMobile())
+        {
+        }
+
+        internal VibrationModule(bool isMobile)
```

Comment:
It's there for testability purposes, but if we're touching raw UWP, not much point in that...